source/ui.js

```diff
@@ -73,22 +73,30 @@ const printCommitLog = async (repoUrl, registryUrl, fromLatestTag, releaseBranch
 	};
 };
 
-const checkIgnoredFiles = async pkg => {
-	const ignoredFiles = await util.getNewAndUnpublishedFiles(pkg);
-	if (!ignoredFiles || ignoredFiles.length === 0) {
+const checkNewFiles = async pkg => {
+	const newFiles = await util.getNewFiles(pkg);
+	if ((!newFiles.unpublished || newFiles.unpublished.length === 0) && (!newFiles.firstTime || newFiles.firstTime.length === 0)) {
 		return true;
 	}
 
-	const message = `The following new files are not already part of your published package:\n${chalk.reset(ignoredFiles.map(path => `- ${path}`).join('\n'))}`;
+	const messages = [];
+	if (newFiles.unpublished.length > 0) {
+		messages.push(`The following new files will not be part of your published package:\n${chalk.reset(newFiles.unpublished.map(path => `- ${path}`).join('\n'))}`);
```

Comment:
I think this message is inaccurate? Maybe it should read:

> The following new files are not already part of your published package

instead?

Reply:
@jessieay I agree! This is a very confusing message! Happens on files which have no apparent reason not to be included!

Reply:
Ok great seems to be fixed in new version. Thanks!